src/main/java/nextstep/subway/ui/LineController.java

```diff
@@ -0,0 +1,47 @@
+package nextstep.subway.ui;
+
+import nextstep.subway.application.LineService;
+import nextstep.subway.dto.LineRequest;
+import nextstep.subway.dto.LineResponse;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+public class LineController {
```

Comment:
사소하지만 해당 api들의 시작은 lines로 시작하고 있어서
위로 빼줘도 좋을 것 같아요 :)

```suggestion
@RestController
@RequestMapping("lines")
public class LineController {
```

Reply:
모든 api들이 lines 으로 시작되어서 말씀하신대로 변경하는 편이 깔끔하겠네요!